src/assets/fabricator/styles/components/layout.css

```diff
@@ -0,0 +1,67 @@
+.f-Layout {
+  height: 100%;
+}
+
+.f-Layout.f-is-active {
+  overflow: hidden;
+}
+
+@media (--large-viewport) {
+  .f-Layout.f-is-active {
+    overflow: auto;
+  }
+}
+
+.f-Layout-body {
+  position: relative;
+  margin: 0;
+}
+
+.f-is-active .f-Layout-body {
```

Comment:
Is `f-Layout` the thing getting the active state here? I'd make it clearer via `.f-Layout.is-active .f-Layout-body`.